packages/ember-glimmer/tests/integration/content-test.js

```diff
@@ -975,10 +975,10 @@ if (!EmberDev.runningProdBuild) {
         userValue: 'width: 42px'
       });
 
-      assert.deepEqual(warnings, [styleWarning]);
+      assert.deepEqual(warnings, [STYLE_WARNING]);
     }
 
-    ['@test specifying `<div style={{{userValue}}}></div>` works properly without a warning'](assert) {
+    ['@htmlbars specifying `<div style={{{userValue}}}></div>` works properly without a warning'](assert) {
```

Comment:
@krisselden Not sure how to handle this case because we have a `TrustingAppendOpcode` here which blindly appends and the context is lost in the compile.